tests/testCoaddPsf.py

```diff
@@ -32,19 +32,10 @@
 import lsst.afw.coord as afwCoord
 import lsst.meas.algorithms as measAlg
 import lsst.pex.exceptions as pexExceptions
-import lsst.pex.logging as logging
 from lsst.afw.geom.polygon import Polygon
 import lsst.utils.tests
 
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace.setVerbosity("meas.algorithms.Interp", verbose)
-    logging.Trace.setVerbosity("afw.detection.Psf", verbose)
```

Comment:
Similar to ``testInterp.py``, there are debug messages from ``afw.math`` but no loggers of these names exist.